src/staticanalysis/bot/static_analysis_bot/cli.py

```diff
@@ -111,11 +137,9 @@ def main(source,
         reporters['phabricator'].setup_api(phabricator_api)
 
     # Load unique revision
-    if source == 'phabricator':
```

Comment:
I wonder if we shouldn't remove all the branches where we check for phab and all class hierarchies related to reporters, since Phab is going to be the only one. (of course, not in this patch and not in the short term ;))

Reply:
https://github.com/mozilla/release-services/issues/1899